src/features/pages/home/our-platforms/index.tsx

```diff
@@ -22,10 +24,16 @@ const OurPlatforms = () => {
             ? '_t_Choose from {{platform_count}} powerful platforms — each designed with your needs in mind._t_'
             : '_t_Choose from {{platform_count}} powerful platforms — designed with you in mind_t_'
     }, [is_eu])
-
+    const [current_widget, setCurrentWidget] = useState<TString>('_t_hey_t_')
     return (
         <Container.Fixed as="section" pt="16x" md={{ padding_block: '40x' }}>
             <Typography.Heading as="h1" size="medium" align="center" mb="10x">
+                <TabMenu
+                    class_name={test}
+                    tab_names={['_t_hey_t_', '_t_you_t_', '_t_watsup_t_']}
+                    current_tab={current_widget}
+                    setCurrentTab={setCurrentWidget}
+                />
```

Comment:
Just example, let's play with it
<img width="1221" alt="Screenshot 2023-05-19 at 16 23 24" src="https://github.com/binary-com/deriv-com/assets/103182473/81e01f97-6e7c-40ab-88be-0ef3e6b1e8c9">

Reply:
example with tab content would be better